numpy/lib/type_check.py

```diff
@@ -576,10 +576,10 @@ def common_type(*arrays):
     type, even if all the arrays are integer arrays. If one of the inputs is
     an integer array, the minimum precision type that is returned is a
     64-bit floating point dtype.
-
-    All input arrays can be safely cast to the returned dtype without loss
-    of information.
-
+	
+    All input arrays except for int64 and uint64 can be safely cast to the
+    returned dtype without loss of information.
+    
```

Comment:
And now you've added trailing spaces / tabs. Could you fix this too?  Sorry to nitpick